custom/static/css/bootstrap-responsive.css

```diff
@@ -997,7 +997,7 @@
   }
   .navbar-inverse .nav-collapse .nav > li > a,
   .navbar-inverse .nav-collapse .dropdown-menu a {
-    color: #999999;
+    color: #444444;
```

Comment:
Whoops, that's what I get for being rushed (e.g. project-wide find/replace...). Will fix and commit